src/components/Form/RemainingInfo/CommercialInfo.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import FormikTextField from "../FormikTextField";
+
+function CommercialInfo() {
```

Comment:
LegalInfo and CommercialInfo do do exactly the same.

You can create single componenent and pass in props valid list. You can store in static.js comercialFieldsToShow and legalFieldsToShow